peerfinder/peerfinder.py

```diff
@@ -185,60 +188,131 @@ def main():
         ix_set = [pdb_to_ixp(ix) for _, ix in ix_dedup.items()]
         netfac_set = [pdb_to_fac(ix) for ix in pdb["data"][0]["netfac_set"]]
         peers.append(pdb_to_peer(pdb["data"][0], ix_set, netfac_set))
+    return(peers)
 
-    if args.ix_only:
-        common_ix_list = fetch_common_ixps(peers)
-        if len(common_ix_list) < 1:
-            print("Didnt find any common IX, exiting...")
-            exit(1)
-        header = []
-        header.append("IX")
+def find_common_points(common_ix_list: List, common_fac_list: List, peers: List) -> Dict:
```

Comment:
Consider a return type of List[IXP] for this method

Reply:
I will refactor this and split it in two functions which can be called independ.